docs/topics/proc_attaching-a-remote-debugger-to-the-application.adoc

```diff
@@ -0,0 +1,33 @@
+
+[#attaching-a-remote-debugger-to-the-application]
+= Attaching a Remote Debugger to the Application
+
+When your application is configured for debugging, attach a remote debugger of your choice to it.
+In this guide, The xref:https://www.redhat.com/en/technologies/jboss-middleware/developer-studio[Red Hat JBoss Developer Studio] is covered, but the procedure is similar when using other programs.
+
+.Prerequisites
+
+* The application running either locally or on OpenShift, and configured for debugging.
```

Comment:
no comma needed since there isn't a new subject

Reply:
I added it for clarity because the sentence could otherwise be understood as _The application running (either locally) or (on OpenShift and configured for debugging)._ The intended way is _The application (running either locally or on Openshift) and configured for debugging. For that reason, I'll take the liberty to keep it.